server/src/utils/fight/fightMethods.ts

```diff
@@ -1776,10 +1781,15 @@ export const playFighterTurn = (
         if (!deflected && block(currentFighter, currentOpponent, 2)) {
           damage = 0;
 
+          // Add evade step
+          fightData.steps.push({
+            a: StepType.Evade,
+            f: opponent.index,
+          });
```

Comment:
Huh?

Reply:
Same for this, new step , changing to index, if you're wondering why all the lines are green for changing, I truly don't know